### PR TITLE
Plotx ploty

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by ericdill** [comment]
This PR looks for 'plotx' and 'ploty' keys in the begin_run_event and triggers replay to set those as the x axis and the y data shown

File: dataportal/replay/scalar/model.py [hunk 1]
```diff
@@ -384,6 +388,47 @@ def update_estimate_target(self, changed):
     def update_normalize(self, changed):
         self.get_new_data_and_plot()
 
+    def set_state(self):
+        plotx = self.header.__dict__.get('plotx')
```
Comment (tacaswell) [inline dataportal/replay/scalar/model.py]: I would use `getattr` here

Comment (ericdill) [inline dataportal/replay/scalar/model.py]: If i use getattr then i need to wrap this in a try/except block

Comment (ericdill) [inline dataportal/replay/scalar/model.py]: Though I now see that there is an optional default argument that getattr takes. i'll update this

**Pull request merged**